src/app/components/Spinner/index.module.scss

```diff
@@ -0,0 +1,15 @@
+.spinner {
+    @apply tw-flex tw-items-center tw-justify-center tw-text-gray-7;
+    svg {
+        min-width: 10px;
+        min-height: 10px;
+        max-width: 100px;
+        max-height: 100px;
+        path {
+            &:first-child {
+                @apply tw-opacity-25;
+                stroke: rgb(92, 112, 128);
```

Comment:
Why do we use a color that is not in our color paletter (in `tailwind.config.js`)? Try to ask tzu if he's ok with that.

Reply:
this color was taken from the blueprintjs/spinner

Reply:
Ok but still, verify it with him pls. I don't dispute the fact it was in blueprintjs that way but since we are building our custom component, I believe we should customize them so they fit our design system.

Reply:
Just change it to the closest matching gray-x

Reply:
yup, I did even better - I used the same color but with 90% transparency, it gives the same effect and looks much more harmonious if you make it colored

Reply:
Looking good @pietro-maximoff :)